Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcessTest.py

```diff
@@ -0,0 +1,99 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import unittest
+from mantid.simpleapi import SANSILLAutoProcess, config, mtd
+
+
+class SANSILLAutoProcessTest(unittest.TestCase):
```

Comment:
I'm not sure if this is needed, validateInputs we can trust, but if we don't then we have to cover all possible permutations which is not feasible.

Reply:
The actual execution takes some time, so is covered in the system tests.

Reply:
Two more thoughts from me on this:
- My default approach on any new algorithm is to implement a unit test. I like the clarity of this because it avoids a grey area about what code we trust\don't trust - and with lots of people joining\leaving the team I think the clarity is helpful
- I think there are lots of unit tests which don't provide 100% coverage of all possible inputs. My view was that partial coverage was better than no coverage though

@gvardany - since you're the gatekeeper on this you probably have the final say but thought I'd come back with those comments at least

Reply:
I agree in general. We should add a unit test that ensures the minimum options run is operational, @gui-co can you add one? But trying to cover validateInputs is simply impractical for this type of master algorithms.